python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type_l3_port_channels.schema.yml

```diff
@@ -0,0 +1,227 @@
+        encapsulation_dot1q_vlan:
+          description: For subinterfaces the dot1q vlan is derived from the interface name by default, but can also be specified.
+          type: int
+          convert_types:
+            - str
+          min: 1
+          max: 4094
+        dhcp_accept_default_route:
+          type: bool
+          default: true
+          description: Accept a default route from DHCP if `ip_address` is set to `dhcp`.
+        enabled:
+          type: bool
+          default: true
+          description: Enable or Shutdown the interface.
+        peer:
+          type: str
+          description: |-
+            The peer device name. Used for description and documentation.
+        peer_port_channel:
+          type: str
+          description: |-
+            The peer device port-channel interface. Used for description and documentation.
+        peer_ip:
+          type: str
+          description: |-
+            The peer device IPv4 address (no mask). Used as default route gateway if `set_default_route` is true and `ip` is an IP address.
+        bgp:
+          type: dict
+          description: |-
+            Enforce IPv4 BGP peering for the peer
+          keys:
+            peer_as:
+              type: str
+              required: true
+              convert_types:
+                - int
+              description: 'BGP AS <1-4294967295> or AS number in asdot notation
+                "<1-65535>.<0-65535>".
+
+                For asdot notation in YAML inputs, the value must be put in
+                quotes, to prevent it from being interpreted as a float number.'
+            ipv4_prefix_list_in:
+              type: str
+              description: |-
+                Prefix List Name. Accept routes for only these prefixes from the peer.
+                Required for wan interfaces.
+            ipv4_prefix_list_out:
+              type: str
+              description: |-
+                Prefix List Name. Advertise routes for only these prefixes.
+                If not specified, nothing would be advertised.
+        ipv4_acl_in:
+          description: |-
+            Name of the IPv4 access-list to be assigned in the ingress direction.
+            The access-list must be defined under `ipv4_acls` and supports field substitution for "interface_ip" and "peer_ip".
+            Required for all WAN interfaces (`wan_carrier` is set) unless the carrier is marked as 'trusted' under `wan_carriers`.
+          type: str
+          convert_types:
+            - int
+        ipv4_acl_out:
+          description: |-
+            Name of the IPv4 Access-list to be assigned in the egress direction.
+            The access-list must be defined under `ipv4_acls` and supports field substitution for "interface_ip" and "peer_ip".
+          type: str
+          convert_types:
+            - int
+        static_routes:
+          description: Configure IPv4 static routes pointing to `peer_ip`.
+          type: list
```

Comment:
Let's add `primary_key: prefix` here and remove the `required` under the prefix key.